src/db/tables/subscribers.ts

```diff
@@ -564,21 +616,25 @@ async function getSubscribersWithUnresolvedBreachesCount() {
   const count = parseInt((await query.count({ count: "*" }))[0].count);
   return count;
 }
+type WithEmailAddresses = SubscriberRow & {
+  email_addresses: EmailAddressRow[];
+};
+
 /* c8 ignore stop */
 
 // Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
 /* c8 ignore start */
-async function joinEmailAddressesToSubscriber(subscriber: SubscriberRow) {
-  if (subscriber) {
```

Comment:
I removed the `if` here and just made sure that callers only call this function to query for related email addresses if they've found a subscriber.